src/question-one/INSTRUCTIONS.md

```diff
@@ -12,4 +12,8 @@ And finally when the results are displayed we will need to see:
 - The start and end date of the job
 - The name of the Contact assigned to the job
 
-Note: We have provided a `service` prop to the component that is defined in /src/service/DataService.js.  It has access to a graphql client and a http request library.  If you wish to fetch data from the server we suggest you do it from there.
\ No newline at end of file
+Note: We have provided a `service` prop to the component that is defined in /src/service/DataService.js.  It has access to a graphql client and a http request library.  If you wish to fetch data from the server we suggest you do it from there.
+
+### What we're looking for
+
+The goal of Question 1 is to see how our candidates think through form submissions and what scenarios that a user will likely encounter, and how to ensure future maintainability of their solution. An example to show future maintainability would be to include some unit tests validating the scenarios that a user is likely to encounter.
```

Comment:
First sentence felt slightly disjointed when I read it. Maybe something like:

The goal of Question 1 is to see how candidates think through form submissions and what scenarios a user is likely to encounter whilst ensuring future maintainability of their solution.

Reply:
Perfect! It certainly didn't read well to me either. Will update